docs/Doxyfile.in

```diff
@@ -1675,10 +1675,12 @@ MATHJAX_FORMAT         = HTML-CSS
 # Content Delivery Network so you can quickly see the result without installing
 # MathJax. However, it is strongly recommended to install a local copy of
 # MathJax from https://www.mathjax.org before deployment.
-# The default value is: https://cdn.jsdelivr.net/npm/mathjax@2.
+# The default value is:
+# in case of MathJax version 2: https://cdn.jsdelivr.net/npm/mathjax@2
+# in case of MathJax version 3: https://cdn.jsdelivr.net/npm/mathjax@3
 # This tag requires that the tag USE_MATHJAX is set to YES.
 
-MATHJAX_RELPATH        = http://cdn.mathjax.org/mathjax/latest
+MATHJAX_RELPATH        = https://cdn.jsdelivr.net/npm/mathjax@2
```

Comment:
NB: Functional change, potentially.

Reply:
`USE_MATHJAX` is currently set to NO, so I think we should be good.

Please also see below.
https://www.mathjax.org/cdn-shutting-down/
https://github.com/doxygen/doxygen/commit/2c7bba0d84ec5616c6ac0c93f53b394fe8f70899

Reply:
Ubuntu 22.04 currently ships `doxygen (1.9.1-2ubuntu2)`. The support for MathJax 3 was added in 1.9.4 in https://github.com/doxygen/doxygen/pull/9218.

Looking at the changelogs https://changelogs.ubuntu.com/changelogs/pool/universe/d/doxygen/doxygen_1.9.1-2ubuntu2/changelog, the support was not backported.